systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1054,4 +1050,33 @@ void waitForDeletion(long time, String namespace) throws Exception {
             throw new Exception("There are some unexpected pods! Cleanup is not finished properly!" + nonTerminated);
         }
     }
+
+    /**
+     * Method for apply Strimzi cluster operator specific Role and CLusterRole bindings for specific namespaces.
+     * @param namespace namespace where CO is deployed
+     * @param clientNamespace namespace, where bindings should be deployed
+     */
+    static void applyRoleBindings(String namespace, String clientNamespace) {
+        // 020-RoleBinding
+        testClassResources.kubernetesRoleBinding("../install/cluster-operator/020-RoleBinding-strimzi-cluster-operator.yaml", namespace, clientNamespace);
+        // 021-ClusterRoleBinding
+        testClassResources.kubernetesClusterRoleBinding("../install/cluster-operator/021-ClusterRoleBinding-strimzi-cluster-operator.yaml", namespace, clientNamespace);
+        // 030-ClusterRoleBinding
+        testClassResources.kubernetesClusterRoleBinding("../install/cluster-operator/030-ClusterRoleBinding-strimzi-cluster-operator-kafka-broker-delegation.yaml", namespace, clientNamespace);
+        // 031-RoleBinding
+        testClassResources.kubernetesRoleBinding("../install/cluster-operator/031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml", namespace, clientNamespace);
+        // 032-RoleBinding
+        testClassResources.kubernetesRoleBinding("../install/cluster-operator/032-RoleBinding-strimzi-cluster-operator-topic-operator-delegation.yaml", namespace, clientNamespace);
+    }
+
+    @BeforeEach
+    void setTestName(TestInfo testInfo) {
+        testName = testInfo.getTestMethod().get().getName();
+    }
+
+    @BeforeAll
+    static void setTestClassName(TestInfo testInfo) {
```

Comment:
Maybe we need to update method name after adding logic for CO

Reply:
I agree, good point.